app/views/tweets/new.html.erb

```diff
@@ -0,0 +1,26 @@
+ <h1>Create Tweet</h1>
+
+ <%= form_for @tweet do |f| %>
+  <% if @tweet.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@tweet.errors.count, "error") %> prohibited this tweet from being saved:</h2>
+
+      <ul>
+      <% @tweet.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= f.hidden_field :user_id %>
```

Comment:
Interesting. Was the original idea to hide the ID of the current user here so you didn't need to write [this](https://github.com/smashingboxes/Ruti-Twitter/pull/3/files#diff-eb437675fc47a98939a642af298c1a19R11) code?

That would be clever but a smart user would change the value of this field and make a tweet looked like it came from any user! That would be bad :(